flutter/lib/src/sentry_flutter.dart

```diff
@@ -27,19 +26,26 @@ typedef FlutterOptionsConfiguration = FutureOr<void> Function(
 mixin SentryFlutter {
   static const _channel = MethodChannel('sentry_flutter');
 
-  /// Initializes the SDK
   static Future<void> init(
     FlutterOptionsConfiguration optionsConfiguration, {
     AppRunner? appRunner,
-    @internal PackageLoader packageLoader = _loadPackageInfo,
-    @internal MethodChannel channel = _channel,
-    @internal PlatformChecker? platformChecker,
+    PackageLoader packageLoader = _loadPackageInfo,
+    MethodChannel channel = _channel,
+    PlatformChecker? platformChecker,
+    bool? enableNative = true,
+    bool? autoInitializeNative = true,
```

Comment:
Is there a way to not use method params but rather reading from `options`? Integrations are run after the SDK is inited, and reading from the options should be enough.